client/src/components/CFooter.style.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+export const Footer = styled.footer`
+  background-color: var(--white);
+  display: flex;
+  flex-direction: column;
+  height: 6rem;
+`;
+export const FooterWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+export const Line = styled.div`
+  display: block;
+  margin: 0 auto;
+  width: 100%;
+  border: 0.12rem solid var(--black);
+  border-radius: 1rem;
+  margin-top: 0.9rem;
+`;
+export const H1 = styled.h1`
```

Comment:
I think it's better if nothing in the footer is a h1, since that's only meant to be used as the main heading of an entire page. Maybe something like an h3 would be better for these, since they're not very important in the structure of the page.